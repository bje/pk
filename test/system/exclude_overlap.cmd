../../src/fieldx -EXX 1
