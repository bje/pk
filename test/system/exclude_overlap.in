aXXXXXa
bXXXXXb
cXXXXXc
dXXXXXd
eXXXXXe
fXXXXXf
