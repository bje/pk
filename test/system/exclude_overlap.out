aXa
bXb
cXc
dXd
eXe
fXf
